tools/BcosAirBuilder/build_chain.sh

```diff
@@ -348,6 +356,33 @@ download_bin()
     chmod a+x ${binary_path}
 }
 
+download_monitor_bin()
+{
+    if [ ! -z "${mtail_binary_path}" ];then
+        LOG_INFO "Use binary ${mtail_binary_path}"
+        return
+    fi
+    if [ "${x86_64_arch}" != "true" ];then exit_with_clean "We only offer x86_64 precompiled fisco-bcos binary, your OS architecture is not x86_64. Please compile from source."; fi
+    mtail_binary_path="bin/${mtail_binary_name}"
+    package_name="${mtail_binary_name}_${compatibility_mtail_version}_Linux_x86_64.tar.gz"
+    if [ -n "${macOS}" ];then
```

Comment:
We can support mtail for macos when using fisco-bcos air.

Reply:
mtail only support linux and windows